### ISSUE TITLE
[Feature request] Suricate support

### ISSUE THREAD

**Issue by Kristinita** [r-stein/sublime-text-keybinding-helper#2]
### 1. Request

It would be nice, if you will make [**Suricate framework**](https://packagecontrol.io/packages/Suricate) support.

### 2. Justification

Example of my Suricate array:

```json
"git-bash": {
    "caption": "Git Bash Here",
    "group": "main.preferences",
    "keys": ["<c>+super+keypad3"],
    "args": {
        "cmd": ["C:\\Program Files\\Git\\git-bash.exe"],
        "working_dir": "${file_path}"
    },
    "call": "Suricate.lib.process.spawn",
    "context_menu": true
},
```

In one array I:

1. add command in command palette,
2. add command in main menu,
3. add command in context menu,
4. add hotkey,

Also:

1. hotkey support all platforms,
2. user can activate and deactivate hotkeys. 

For more information see [**Suricate documentation**](https://sublime-suricate.readthedocs.io/en/latest/quick-guide/). Suricate it is more convenient for me than the native Sublime Text work with commands.

### 3. Desirable behavior

Exactly the same behavior for `suricate-profile` files like you make for `sublime-keymap` files.

Thanks.

**Comment by r-stein** [comment]
This sounds reasonable to add. I will have a look at it.

**Comment by r-stein** [comment]
I have released a new version `2.1.0`, which supports the suricate package.
